src/css/app.css

```diff
@@ -1,33 +1,39 @@
-@font-face {
-    font-display: swap;
-    font-family: 'Source Sans Pro';
-    font-style: normal;
-    font-weight: 300;
-    src: url('../fonts/sourcesanspro-light-webfont.woff2') format('woff2');
-}
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
 
-@font-face {
-    font-display: swap;
-    font-family: 'Source Sans Pro';
-    font-style: normal;
-    font-weight: 400;
-    src: url('../fonts/sourcesanspro-regular-webfont.woff2') format('woff2');
-}
+@layer base {
+    @font-face {
+        font-display: swap;
+        font-family: 'Source Sans Pro';
+        font-style: normal;
+        font-weight: 300;
+        src: url('../fonts/sourcesanspro-light-webfont.woff2') format('woff2');
+    }
 
-@font-face {
-    font-display: swap;
-    font-family: 'Source Sans Pro';
-    font-style: normal;
-    font-weight: 600;
-    src: url('../fonts/sourcesanspro-semibold-webfont.woff2') format('woff2');
-}
+    @font-face {
+        font-display: swap;
+        font-family: 'Source Sans Pro';
+        font-style: normal;
+        font-weight: 400;
+        src: url('../fonts/sourcesanspro-regular-webfont.woff2') format('woff2');
+    }
 
-@font-face {
-    font-display: swap;
-    font-family: 'Source Sans Pro';
-    font-style: normal;
-    font-weight: 700;
-    src: url('../fonts/sourcesanspro-bold-webfont.woff2') format('woff2');
+    @font-face {
+        font-display: swap;
+        font-family: 'Source Sans Pro';
+        font-style: normal;
+        font-weight: 600;
+        src: url('../fonts/sourcesanspro-semibold-webfont.woff2') format('woff2');
+    }
+
+    @font-face {
+        font-display: swap;
+        font-family: 'Source Sans Pro';
+        font-style: normal;
+        font-weight: 700;
+        src: url('../fonts/sourcesanspro-bold-webfont.woff2') format('woff2');
+    }
 }
 
 body {
```

Comment:
@joshuapease I think `body` and `strong` should also be wrapped in `@layer base { ... }`

Reply:
Good catch! Resolved!